spec/rails_helper.rb

```diff
@@ -77,6 +77,7 @@ def call(env)
 
 require Rails.root.join("spec/system/page_objects/pages/base.rb")
 require Rails.root.join("spec/system/page_objects/modals/base.rb")
+require Rails.root.join("spec/system/page_objects/components/base.rb")
```

Comment:
Maybe we can replace L78-L80 with this so there's no surprise if we add a new page object type?

```ruby
Dir[Rails.root.join("spec/system/page_objects/**/base.rb")].each { |f| require f }
```

or have the subclasses `require_relative` their base?